### ISSUE TITLE
Failed To Load Agora Third Party Plugin

### ISSUE THREAD

**Issue by ordyfarmer02** [AgoraIO-Community/Agora-Unreal-SDK#19]
![](https://i.imgur.com/wotjCkg.png)

Thank you for read this.

Engine Version : 4.27
Platform : Window 64bit

I packaged the sample project.
The packaging was successful, but i got a error message like above.

dlls are already in WindowsNoEditor\API_Example\Binaries\Win64
What should i do?

**Comment by alexhajdu** [comment]
As stated in the log, you need to provide the DLL. Just copy the directory, where the `agora_rtc_sdk.dll` resides into the directory with your executable.

```
Project
|-- AgoraPlugin\Source\ThirdParty\Agora\x64\Release\agora_rtc_sdk.dll
| --project.exe

```

**Comment by ordyfarmer02** [comment]
> 

@alexhajdu Thank you for reply. But would you explain more specific?
I moved the AgoraPlugin folder in the directory where the executable file belongs like below, but the same error message keep coming out.

![](https://i.imgur.com/qXnPz5b.png)

and dlls already exist in Binaries folder

![](https://i.imgur.com/K1dKjiU.png)

~~I don't know which directory to put the dlls in exactly.~~

After move AgoraPlugin folder into WindowsNoEditor\API_Example\Plugins\, it works properly.
Thank you.

**Comment by alexhajdu** [comment]
Yes, sorry, was not super specific, but you figured it out. Glad it works
